regression-test/suites/external_table_p0/jdbc/non_connection_pool/test_oracle_jdbc_catalog_no_pool.groovy

```diff
@@ -0,0 +1,69 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Also need to add case to test `alter property`

Reply:
Prohibit altering property